site/app/templates/GlobalHeader.twig

```diff
@@ -263,7 +263,7 @@
                         {% endif %}
                         {{ message.error | nl2br }}
                     </span>
-                    <a class="fas fa-times key_to_click" tabindex="0" onClick="removeMessagePopup('{{ message.type }}-{{ message.key }}');"></a>
+                    <a class="fas fa-times key_to_click" tabindex="0" id = "remove_popup" onClick="removeMessagePopup('{{ message.type }}-{{ message.key }}');"></a>
```

Comment:
Minor nitpick to match the style of the rest of the codebase:
```suggestion
                    <a class="fas fa-times key_to_click" tabindex="0" id="remove_popup" onClick="removeMessagePopup('{{ message.type }}-{{ message.key }}');"></a>
```

The same comment applies elsewhere in this PR.  Apply throughout.